storybook/src/admin/error/inlineError/InlineError.stories.tsx

```diff
@@ -0,0 +1,120 @@
+import { InlineError } from "@comet/admin";
+import { Clear, CometColor, InfoFilled, Reload, RemoveFilled, WarningSolid } from "@comet/admin-icons";
+import { Box, Button } from "@mui/material";
+import type { Meta, StoryObj } from "@storybook/react";
+
+type Story = StoryObj<typeof InlineError>;
+
+const config: Meta<typeof InlineError> = {
```

Comment:
Should we add some information when this component should be used here?

Reply:
We can add this in the future if needed imo.